examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
ok last comment before I actually sign off because this is bugging me. If I follow the code for `plot_compute_raw_data_spectrum` up until just after the notch filter, then run the following:

```python
plt.psd(raw._data[picks[1]], Fs=raw.info['sfreq'])
plt.psd(raw.copy().apply_proj()._data[picks[1]], Fs=raw.info['sfreq'], color='r')
```
Then I get the following output (red == after apply_proj, and note this is only for a single channel):

![image](https://cloud.githubusercontent.com/assets/1839645/12062640/eec49862-af65-11e5-9b8a-963129330cc3.png)

ok I'm super tired and probably making stupid mistakes at this point, so I'll deal with this tomorrow